fe/fe-core/src/main/java/com/starrocks/task/AgentTask.java

```diff
@@ -157,7 +158,14 @@ public boolean isFinished() {
         return isFinished;
     }
 
+    public void setSubmitted(boolean isSubmitted) {
+        this.isSubmitted = isSubmitted;
+    }
+
     public boolean shouldResend(long currentTimeMillis) {
+        if (!isSubmitted) {
```

Comment:
Sometimes tasks should be resent because there will be a failure in the backends. But for this change, it won't be resent anymore, it will cause other issues.